tests/base/develop_pipelines.py

```diff
@@ -97,9 +97,9 @@ def run_model_test(
 
 def run_prediction(trained_model, dataloader, dp=False, min_acc=0.25):
     if isinstance(trained_model, BoringModel):
-        return _boring_model_run_prediction(trained_model, dataloader, dp, min_acc)
+        return _boring_model_run_prediction(trained_model, dataloader, min_acc)
     else:
-        return _eval_model_template_run_prediction(trained_model, dataloader, dp, min_acc)
+        return _eval_model_template_run_prediction(trained_model, dataloader, dp, min_acc=0.50)
```

Comment:
Why do we keep this one?

Reply:
still used in other tests. we can remove all eval model related things once all the tests are updated with BoringModel.